services/QuillLMS/spec/factories/translation_mappings.rb

```diff
@@ -18,4 +18,8 @@
     english_text { create(:english_text) }
     field_name { 'description' }
   end
+
+  factory :translation_mapping_with_translation, parent: :translation_mapping do
+    english_text { create(:english_text, :with_translated_text)}
```

Comment:
Again, this doesn't really matter, but you've got a space after the left curly brace, but not before the right curly brace.

Reply:
I will make a rubocop for this and put it in another PR